### PR TITLE
Support generic type modifiers

### LINKED ISSUE #2: Support numeric(..) and character(..)
Hi! Awesome gem :-).

I was trying it out on one of my tables and I noticed it doesn't yet handle `numeric(20,3)` or `character(1)` columns:

```
pg_column_byte_packer.rb:115:in `ordering_key_for_column': Got sql_type: `numeric(20,3)` and type_schema: `pg_catalog` but was unable to find entry in pg_type. (ArgumentError)
```

I'm using a call to `PgColumnBytePacker::PgDump.sort_columns_for_definition_file` in my test.

### PR DESCRIPTION
**Pull request by jcoleman**
Fixes #2.

### CODE REVIEW

File: lib/pg_column_byte_packer.rb [hunk 1]
```diff
@@ -84,6 +122,14 @@ def self.ordering_key_for_column(connection:, name:, sql_type:, type_schema: nil
             4
           end
         end
+      when /\Abit varying\(\d+\)/
```
Comment (JonMR) [inline lib/pg_column_byte_packer.rb:125]: This is an interesting case. I do wonder if it's worth special casing. I guess the code isn't too bad once you understand the reasoning. Might be worth a comment in the code to explain it.

**Review by JonMR — APPROVED** [review]
LGTM.

File: lib/pg_column_byte_packer.rb (hunk 1 above)
Comment (jcoleman) [reply lib/pg_column_byte_packer.rb:125]: I think I'll add a comment separately since it should also apply to `character varying` above.

**Comment by jcoleman** [comment]
I canceled the duplicative builds (both travis-ci.org and travis-ci.com were running against the repo).

**Pull request merged**